From 08537add6f980c895b649185df8a266c724a70bb Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Tue, 12 Dec 2017 05:10:57 -0600
Subject: [PATCH] "0001-liblog-fix-crash-in-loading-stagefright-hw-ext-s
 _______________________________

*Previous From 7f685bdd152fe9bfd2f9f3ee8bc6d1ab500b42c6 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Wed, 15 Nov 2017 08:21:50 -0600
*Previous Subject: [PATCH] liblog - fix crash in loading stagefright hw ext's
*
*Change-Id: Ic81ce83c6d76e3634bd3a5f6770fd3f4178ac63d"
---
 liblog/logger_write.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/liblog/logger_write.c b/liblog/logger_write.c
index d322c0f..aef4d54 100644
--- a/liblog/logger_write.c
+++ b/liblog/logger_write.c
@@ -52,6 +52,7 @@ static enum {
 } g_log_status = kLogUninitialized;
 
 static int check_log_uid_permissions() {
+#if 0
 #if defined(__ANDROID__)
   uid_t uid = __android_log_uid();
 
@@ -90,6 +91,7 @@ static int check_log_uid_permissions() {
     }
   }
 #endif
+#endif
   return 0;
 }
 
-- 
2.7.4

